### PR TITLE
/8ball, /getafk and /getafks commands

### PR DESCRIPTION
**Pull request by Pyraqq**

### CODE REVIEW

File: server/client_manager.py [hunk 1]
```diff
@@ -334,14 +334,18 @@ def get_area_info(self, area_id, mods):
                 area.Locked.SPECTATABLE: '[SPECTATABLE]',
                 area.Locked.LOCKED: '[LOCKED]'
             }
-            info += f'[{area.abbreviation}]: [{len(area.clients)} users][{area.status}]{lock[area.is_locked]}'
-
+            if afk_check == True:
```
Comment (oldmud0) [inline server/client_manager.py]:
```suggestion
            if afk_check:
```

File: server/client_manager.py [hunk 2]
```diff
@@ -314,7 +314,7 @@ def send_area_list(self):
                     msg += ' [*]'
             self.send_ooc(msg)
 
-        def get_area_info(self, area_id, mods):
+        def get_area_info(self, area_id, mods, afk_check):
             """
             Get information about a specific area.
```
Comment (oldmud0) [inline server/client_manager.py:319]: need to document `:param afk_check:`

File: server/client_manager.py [hunk 3]
```diff
@@ -334,14 +334,18 @@ def get_area_info(self, area_id, mods):
                 area.Locked.SPECTATABLE: '[SPECTATABLE]',
                 area.Locked.LOCKED: '[LOCKED]'
             }
-            info += f'[{area.abbreviation}]: [{len(area.clients)} users][{area.status}]{lock[area.is_locked]}'
-
+            if afk_check == True:
+                list1 = area.afkers
```
Comment (oldmud0) [inline server/client_manager.py]: if there is a list1 where is the list2?

use more descriptive variable names

File: server/client_manager.py [hunk 4]
```diff
@@ -361,7 +365,7 @@ def get_area_info(self, area_id, mods):
                     info += f' ({c.ipid}): {c.name}'
             return info
 
-        def send_area_info(self, area_id, mods):
+        def send_area_info(self, area_id, mods, afk_check):
             """
             Send information over OOC about a specific area.
```
Comment (oldmud0) [inline server/client_manager.py:371]: same problems here

File: server/client_manager.py [hunk 5]
```diff
@@ -374,18 +378,34 @@ def send_area_info(self, area_id, mods):
...
                 info = '\n== Area List =='
                 for i in range(len(self.server.area_manager.areas)):
-                    if len(self.server.area_manager.areas[i].clients
-                           ) > 0 or len(
+                    if afk_check == True:
+                        client_list1 = self.server.area_manager.areas[i].afkers
+                        get_area1 = self.get_area_info(i, mods, True)
+                    else:
+                        client_list1 = self.server.area_manager.areas[i].clients
+                        get_area1 = self.get_area_info(i, mods, False)
```
Comment (oldmud0) [inline server/client_manager.py]: nicer version:
```python
client_list = self.server.area_manager.areas[i]
if afk_check:
    client_list = client_list.afkers
else:
    client_list = client_list.clients
area_info = self.get_area_info(i, mods, afk_check)
```

File: server/commands/roleplay.py [hunk 6]
```diff
@@ -213,3 +219,23 @@ def ooc_cmd_coinflip(client, arg):
...
+        arg = arg.strip()
+        if len(arg) == 0:
+            raise ArgumentError('You need to ask a question')
+        rolla_reload(client.area)
+        ability_dice = client.area.ability_dice['8ball']
+        client.area.broadcast_ooc('{} asked a question: {} and the answer is: {}.'.format(
+            client.char_name, arg, rolla(ability_dice)[2]))
+    except Exception as error:
+        error_mes = ", ".join([str(s) for s in error.args])
+        raise ClientError(error_mes)
```
Comment (oldmud0) [inline server/commands/roleplay.py]: what is this, get it out of here

**Comment by oldmud0** [comment]
is it tested

**Comment by Pyraqq** [comment]
Yes and works.

**Comment by oldmud0** [comment]
ok lobo

**Pull request merged**